<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hkl_viewer_setup_dialog</class>
 <widget class="QDialog" name="hkl_viewer_setup_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="gbox_roi_info">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>451</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Optional Setup</string>
   </property>
   <widget class="QLineEdit" name="le_roi_config">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>431</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid black;
border-radius:3;

</string>
    </property>
    <property name="text">
     <string>/home/beams0/JULIO.RODRIGUEZ/Lab_Software/area_det_PVA_viewer/pv_configs/metadata_pvs.toml</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_browse">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>110</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
font: 11pt &quot;Sans Serif&quot;;
border: 1px solid #8f8f91;
border-radius: 3px;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #f6f7fa, stop: 1 #dadbde);
}
QPushButton::pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_create">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
font: 11pt &quot;Sans Serif&quot;;
border: 1px solid #8f8f91;
border-radius: 3px;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #f6f7fa, stop: 1 #dadbde);
}
QPushButton::pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>Create</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_edit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>110</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
font: 11pt &quot;Sans Serif&quot;;
border: 1px solid #8f8f91;
border-radius: 3px;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #f6f7fa, stop: 1 #dadbde);
}
QPushButton::pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_roi_config">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>34</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Config</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gb_viewer_setup">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>451</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Required Setup</string>
   </property>
   <widget class="QLineEdit" name="le_input_channel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>421</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid black;
border-radius:3;
</string>
    </property>
    <property name="text">
     <string>processor:1:analysis</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_input_channel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Input Channel</string>
    </property>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="btn_accept_reject">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>330</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>le_input_channel</tabstop>
  <tabstop>btn_browse</tabstop>
  <tabstop>le_roi_config</tabstop>
  <tabstop>btn_edit</tabstop>
  <tabstop>btn_create</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btn_accept_reject</sender>
   <signal>accepted()</signal>
   <receiver>hkl_viewer_setup_dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>575</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_accept_reject</sender>
   <signal>rejected()</signal>
   <receiver>hkl_viewer_setup_dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>575</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
